stock_prices/stock_prices.py

```diff
@@ -1,15 +1,23 @@
-#!/usr/bin/python
 
 import argparse
 
+
 def find_max_profit(prices):
-  pass
+  low = prices[0]
```

Comment:
Amazing job here! This is incredibly clean and simple. I love this. This is basically a recursive solution. I'm proud of your Laryna!